code/modules/mob/living/basic/space_fauna/spider/giant_spider/giant_spider_variants.dm

```diff
@@ -18,6 +47,41 @@
 	player_speed_modifier = -3.1
 	menu_description = "Fast spider variant specializing in catching running prey and toxin injection, but has less health and damage."
 
+/mob/living/basic/giant_spider/hunter/Initialize(mapload)
+	. = ..()
+
+	AddElement(/datum/element/web_walker, /datum/movespeed_modifier/hunter_web)
+
+/**
+ * ### Spider Scout
+ * A subtype of the giant spider which is faster, has phermone detection, but less health and damage.
```

Comment:
Just say "can see mobs through walls" instead of pheromone detection so it is clear what you actually mean.